packages/clerk-js/src/ui/signUp/SignUpContinue.tsx

```diff
@@ -65,12 +65,14 @@ function _SignUpContinue(): JSX.Element | null {
   const hasEmail = !!formState.emailAddress.value;
   const hasVerifiedExternalAccount = signUp.verifications?.externalAccount?.status == 'verified';
   const hasVerifiedWeb3 = signUp.verifications?.web3Wallet?.status == 'verified';
+  const isProgressiveSignUp = userSettings.signUp.progressive;
```

Comment:
🙃 
This should higher up IMO, will help with understanding a bit better the flow. It is also used on the commIdentifier.